lib/AST/Decl.cpp

```diff
@@ -46,6 +46,7 @@
 #include "swift/Basic/Range.h"
 #include "swift/Basic/StringExtras.h"
 #include "swift/Basic/Statistic.h"
+#include "swift/SIL/SILLinkage.h"
```

Comment:
AST should not depend on SIL. Instead, make this a global function in lib/SIL/SIL.cpp.

Reply:
Ok, that makes sense. I'll rework this.

Reply:
I force updated this change and moved the implementation to lib/SIL/SIL.cpp. include/SIL/SILModule.h seemed to be the best place for this declaration to live, but would it make sense for this to be a member function on SILModule directly?